docs/source/overview.rst

```diff
@@ -1,19 +1,57 @@
 .. _code_structure:
 
 ######################################
-Code structure
+Overall design
 ######################################
 
-Most of :ref:`operations <operations>` in Workflow take in (optional) and return (mandatory) an ASE's ``Atoms`` object. These inputs and outputs are abstracted in ``ConfigSet`` and ``OutputSpec`` classes.  ``ConfigSet`` may be initialised with a list(s) of ``Atoms``, by providing one or multiple ``.xyz`` filenames or with a reference to an `ABCD database <https://github.com/libAtoms/abcd>`_. Similarly, returned configurations can be held in memory and subsequently used via ``OutputSpec.to_ConfigSet()``, written to a single or multiple files or uploaded to an ABCD database. This way, an operation may iterate over a ``ConfigSet`` and write ``Atoms`` to ``OutputSpec``, regardless of how the input configs were supplied or how or where to the output configs are going to be collected.
+The main goal of the workflow is make it easy operate on sets of input atomic
```

Comment:
"main goal of Workflow" or "main goal of the Workflow package"? Also - "is **to** make it easy **to** operate"?